### ISSUE TITLE
ethermint: command not found

### ISSUE THREAD

**Issue by ryok-0319** [cosmos/ethermint-archive#479]
I'm using MacOS High Sierra.

I installed ethermint by `go get github.com/tendermint/ethermint`
And `cd` into `ethermint` and run `make install`
It returns 
```
go install -tags netgo -ldflags "-X github.com/cosmos/ethermint/version.GitCommit=c06a3f9" ./cmd/emintd
go install -tags netgo -ldflags "-X github.com/cosmos/ethermint/version.GitCommit=c06a3f9" ./cmd/emintcli]
```
and apparently there is no error.

However, when I do `ethermint version`, it returns `zsh: command not found: ethermint`

go version :
`go1.10.1 darwin/amd64`

echo $GOPATH : 
`/Users/ryo/go`

pwd : 
`/Users/ryo/go/src/github.com/tendermint/ethermint`

**Comment by zramsay** [comment]
do you have `GOPATH` on `PATH`? https://github.com/tendermint/tendermint/wiki/Setting-GOPATH

**Comment by zramsay** [comment]
try:
```
go get github.com/cosmos/ethermint
cd $GOPATH/src/github.com/cosmos/ethermint
make all
```

and the binary names are:
```
emintd
emintcli
```

**Comment by ryok-0319** [comment]
In my `.zshrc`, I have `GOPATH` and `PATH` as follows.
```
# Go path
export GOPATH=/Users/ryo/go
export GOBIN=/Users/ryo/go/bin
export PATH=/bin:/Users/ryo/.rbenv/shims:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Postgres.app/Contents/Versions/latest/bin:/Users/ryo/.nodebrew/current/bin:/Users/ryo/go/bin
```

And I tried as you commented above.
```
go get github.com/cosmos/ethermint
cd $GOPATH/src/github.com/cosmos/ethermint
make all
```

However, `ethermint version` still returns `zsh: command not found: ethermint` :(

**Comment by zramsay** [comment]
see the second half of my previous comment:

the binary names are:
```
emintd
emintcli
```

That said, you'll notice that these binaries don't actually do anything - there is currently no CLI

**Comment by JakeLin** [comment]
Hi @zramsay, when I run 

```
emintd
emintcli
```

Nothing happens, please also see #500 , do you know why? thanks

**Comment by alexanderbez** [comment]
@JakeLin those binaries do not work because Ethermint v2 is yet to be completed. Even a minimal MVP. We expect to have something developers can use by mid January.

**Comment by JakeLin** [comment]
@alexanderbez thanks for your update

**Comment by ghost** [comment]
@alexanderbrez if you provide some direction for development ? And will you consider community to contribute to ethermint development? I am keen to get involved but not sure how can we map evm in a x module

**Comment by alexanderbez** [comment]
@bobio2018 we're certainly very open to getting community help and involvement! You can take a look at the current PR #496 to see the basis of the EVM module.

I think we need to further solidify a development roadmap and create further Github issues.

**Comment by ghost** [comment]
@alexanderbez thank you alex, is there a chat channel myself and Jake can join to discuss in more details? We are a small working group actively participating in blockchain space.

**Comment by alexanderbez** [comment]
@bobio2018 yeah, I think if you can sync up with @jackzampolin, he could tell you more details. We do have a riot chat as well if you want to introduce yourselves there initially.

**Comment by jackzampolin** [comment]
@bobio2018 shoot me an email! jack@tendermint.com and let's chat!

**Comment by ghost** [comment]
@jackzampolin email sent, thanks Jack and Alex!

**Comment by grandbaiyi** [comment]
Hi @alexanderbez, when will the ethermint 2.0 release? It seems the development work already stop.
